src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,49 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private static final String COMA = ",";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        int[] newInfoForSupplyAndBuy = readFile(fromFileName);
+        writeFile(toFileName, newInfoForSupplyAndBuy);
+    }
 
+    private void writeFile(String toFileName, int[] information) {
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(toFileName))) {
+            String stringBuilder = SUPPLY + COMA + information[0] + System.lineSeparator()
```

Comment:
create separate method generateReport, replace logic of making report there, use StringBuilder with methods append() to avoid creating new String object each time during concatenation